inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java

```diff
@@ -778,4 +825,53 @@ private List<StreamSourceEntity> fetchTask(TaskRequest request) {
         return taskLists;
     }
 
+    private ConfigResult reloadModuleConfigs(ConfigRequest request) {
```

Comment:
reloadModuleConfigs doesn't seem like it should have been triggered by an agent request

Reply:
Done.